scripts/onboarding/templates/arc-k8s-extension-provision-msi-auth/existingClusterOnboarding.json

```diff
@@ -0,0 +1,202 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",

```

Comment:
MSI auth template & parameter file needs to be updated accordingly with changes in Kaveesh merged and once his PR merged.